src/frontend/components/NoteCard/NoteCard.jsx

```diff
@@ -1,10 +1,36 @@
 import React from "react";
 import "./NoteCard.css";
 import parse from "html-react-parser";
-const NoteCard = ({ noteData }) => {
+const NoteCard = ({ note }) => {
+  const { title, content, createdAt } = note;
   return (
     <div className="ql-editor note-card m-4 rounded-md">
-      {parse(`${noteData}`)}
+      <h2>{title}</h2>
+      {parse(`${content}`)}
+      <div className="d-flex justify-between align-center pt-4">
+        <div className="d-flex flex-col">
+          <span>{new Date(createdAt).toLocaleDateString("in-IN")}</span>
+          <span>
+            {new Date(createdAt)
+              .toLocaleTimeString()
+              .replace(/([\d]+:[\d]{2})(:[\d]{2})(.*)/, "$1$3")}
```

Comment:
Suggestion: you can use a separate util function for formatting dates, this will increase code readability.